MUXSDKStats/MUXSDKStats/MUXSDKStats.m

```diff
@@ -221,6 +221,7 @@ + (void)videoChangeForPlayer:(nonnull NSString *)name withPlayerData:(nullable M
             if (playerData) {
                 [_customerPlayerDataStore setPlayerData:playerData forPlayerName:name];
             }
+            [player prepareForAvQueuePlayerNextItem];
```

Comment:
Oh I see, you actually don't check for `AVQueuePlayer` out here, you check inside of `prepareForAvQueuePlayerNextItem`

Reply:
yeah this top level MUXSDKStats class doesn't have access to the underlying `player` instance... And that felt more like the responsibility of the binding to handle internally

Reply:
💯